vllm/_custom_ops.py

```diff
@@ -1274,7 +1274,9 @@ def scaled_fp8_quant(
             scale = torch.zeros(1, device=input.device, dtype=torch.float32)
             torch.ops._C.dynamic_scaled_fp8_quant(output, input, scale)
     else:
-        assert scale.numel() == 1
+        # num_token_padding not implemented for this case
+        assert (scale.numel() == 1 and num_token_padding
+                is None), f"{scale.shape} {num_token_padding}"
```

Comment:
This should just be `assert scale.numel() == 1` -- `num_token_padding` should be fine for this case